saleor/graphql/translations/types.py

```diff
@@ -146,13 +151,20 @@ def resolve_product_variant(root: product_models.ProductVariant, info):
 
     @staticmethod
     def resolve_attribute_values(root: product_models.ProductVariant, _info):
-        translatable_values = []
-        for assignment in root.attributes.all():
-            attr = assignment.attribute
-            if attr.input_type in AttributeInputType.TRANSLATABLE_ATTRIBUTES:
-                value = assignment.values.first()
-                translatable_values.append(value)
-        return translatable_values
+        def func(attributes):
+            translatable_values = []
+            for assignment in attributes:
+                attr = assignment["attribute"]
+                if attr.input_type in AttributeInputType.TRANSLATABLE_ATTRIBUTES:
+                    value = assignment["values"][0]
+                    translatable_values.append(value)
+            return translatable_values
+
+        return (
+            SelectedAttributesByProductVariantIdLoader(_info.context)
```

Comment:
```suggestion
            SelectedAttributesByProductVariantIdLoader(info.context)
```
info is no longer unused so remove `_` from it